apps/client/app/routes/_index.tsx

```diff
@@ -1,21 +1,45 @@
-import type { MetaFunction } from "@remix-run/node";
-import EmailPreviewer from "../components/EmailPreviewer";
+import Navbar, {  } from "~/components/navbars/navbar";
+import { ChevronRightIcon } from "@heroicons/react/24/solid";
+import { NavbarLinks } from "../../models/navbarLinks.model";
 
-export const meta: MetaFunction = () => {
-  return [
-    { title: "New Remix App" },
-    { name: "description", content: "Welcome to Remix!" },
-  ];
-};
+const links: NavbarLinks[] = [
+  {
+    path: "/login",
+    label: "Sign in"
+  },
+  {
+    path: "/signup",
+    label: "Sign up"
+  },
+  {
+    path: "/docs",
+    label: "Docs"
+  },
+  {
+    path: "/emails",
+    label: "Emails"
+  }
+]
```

Comment:
I don't think there is a need to have the possibility to configure links in the navbar (or sidebar)
- They will be mostly static
- More work is required to change/update arrays like this in components

what do you think?

Reply:
Yeah I left the ability to pass the NavLinks as props in case we want tot reuse the component somewhere. But know Im not even sure if that will ever happen. I will simplify it it for now.